### PR TITLE
Remove entries already covered by support expressions

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

**Comment by autoantwort** [comment]
Only baseline regressions in the ci

**Comment by JonLiu1993** [comment]
@autoantwort, thanks for your PR, please remove conflict file.

**Comment by JonLiu1993** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -656,22 +595,7 @@ libgpg-error:arm64-android=fail
...
 # Missing system libraries on linux to run/prepare autoconf
 libgpod:x64-linux=fail
-libgpod:x64-osx=fail
-libevhtp:x86-windows=fail
-libevhtp:x64-windows=fail
-libevhtp:x64-windows-static=fail
-libevhtp:x64-windows-static-md=fail
-libevhtp:x64-uwp=fail
-libevhtp:arm64-windows=fail
-libevhtp:arm-uwp=fail
```
Comment (JonLiu1993) [inline scripts/ci.baseline.txt:666]: @autoantwort, thanks for the PR, I noticed that libevhtp supports "!windows", why did you remove this code?
````
libevhtp:x64-uwp=fail
libevhtp:arm-uwp=fail
````

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
I have compared the `supports` expression in vcpkg.json.

**Review by BillyONeal — APPROVED** [review]
